### ISSUE TITLE
Ability to have attributes sorted

### ISSUE THREAD

**Issue by brianjenkins94** [htmlhint/HTMLHint#309]
**Is your feature request related to a problem? Please describe.**
Unsorted HTML attributes are annoying to look through but even more annoying to fix. It would be neat if HTMLHint could automatically order attribute tags.

**Describe the solution you'd like**
When linted, HTML attributes should be sorted in some (ideally configurable) order. Like so:

```html
<div id="one" class="red"></div>
<div class="blue" id="two"></div>
```

Should become:

```html
<div id="one" class="red"></div>
<div id="two" class="blue"></div>
```

**Describe alternatives you've considered**
\--

**Additional context**
I don't exactly agree with this sort order but it illustrates the point:

http://codeguide.co/#html-attribute-order

**Comment by nebulousdog** [comment]
I think this got implemented in https://github.com/htmlhint/HTMLHint/pull/332?

**Comment by nebulousdog** [comment]
@brianjenkins94 I think this got implemented in #332.
